run-ccsp2-build

```diff
@@ -19,6 +19,7 @@ export METRICS_UTILITY_REPORT_RHN_LOGIN="test_login"
 export METRICS_UTILITY_REPORT_SKU="MCT3752MO"
 export METRICS_UTILITY_REPORT_SKU_DESCRIPTION="EX: Red Hat Ansible Automation Platform, Full Support (1 Managed Node, Dedicated, Monthly)"
 
+/usr/bin/time --format='\n\nmemory %MK\ncpu %P\ntime %es\n\n' \
```

Comment:
this is now entirely sufficient, we don't need to upload to documents.

We'll be doing Observability integration later, so we'll deploy e.g. Tempo locally to observe where the report spends time.

Reply:
also, we can just insert output of this to any PR that can have performance implications. Just running it before and after the PR. So this is awesome!!!